package.json

```
@@ -53,6 +51,7 @@
     "prettier": "3.6.2",
     "prettier-plugin-astro": "0.14.1",
     "sharp": "0.34.4",
+    "typescript-eslint": "^8.46.1",
```

Should we pin the version?

```suggestion
    "typescript-eslint": "8.46.1",
```